src/AppBundle/Entity/User.php

```diff
@@ -11,11 +11,38 @@
  */
 class User extends BaseUser
 {
-    const GRADE_RECRUIT = 'recruit';
-    const GRADE_SOLDIER = 'soldier';
-    const GRADE_LIEUTENANT = 'lieutenant';
-    const GRADE_CAPTAIN = 'captain';
-    const GRADE_COLONEL = 'colonel';
+    /**
+     * English names based upon (consulted on 2017-03-01):
+     * https://en.wikipedia.org/wiki/Military_ranks_of_the_Swiss_Armed_Forces
+     *
+     * Nonexistant equivalent or multiple equivalents have been adapted to avoid collisions
+     */
+
+    const RANK_RECRUIT = 'recruit';
+    const RANK_PRIVATE = 'private';
+    const RANK_APPOINTEE = 'appointee';
+    const RANK_PRIVATE_FIRST_CLASS = 'private first class';
```

Comment:
As value of constant will be translation key, it's better to replace space by ``_``